src/core/connection.c

```diff
@@ -7170,6 +7170,31 @@ QuicConnParamGet(
         break;
     }
 
+    case QUIC_PARAM_CONN_ORIG_DEST_CID:
+        if (Connection->OrigDestCID == NULL) {
```

Comment:
Still need to do this after the length check.

Reply:
Interesting, so even if Connection->OrigDestCID is NULL, we allow the user to ask "how big is OrigDestCID?" In that case, we tell them 0. I can refactor.

Reply:
Ah, sorry. I mixed this up. This is right as is.